docs/topics/multiplatform/multiplatform-direct-integration.md

```diff
@@ -1,11 +1,20 @@
 [//]: # (title: Direct integration)
 
+<tldr>
+   This is a local integration method that can work for you if:<br/>
```

Comment:
```suggestion
   This local integration method can work for you if:<br/>
```

Reply:
Yes, the single sentence was trying to do a bit too much on its own. And now there's a proper cadence, nice 👍